.github/workflows/yaks-tests.yaml

```diff
@@ -42,7 +42,7 @@ concurrency:
 
 env:
   YAKS_VERSION: 0.16.0
-  YAKS_RUN_OPTIONS: "--timeout=15m --local -e YAKS_CAMELK_MAX_ATTEMPTS=10 -e YAKS_JBANG_CAMEL_VERSION=4.0.0 -e YAKS_JBANG_KAMELETS_VERSION=4.0.0-SNAPSHOT -e YAKS_JBANG_KAMELETS_LOCAL_DIR=../../../kamelets -e YAKS_KAMELET_API_VERSION=v1alpha1"
+  YAKS_RUN_OPTIONS: "--timeout=15m --local -e YAKS_CAMELK_MAX_ATTEMPTS=10 -e YAKS_JBANG_CAMEL_VERSION=4.0.0 -e YAKS_JBANG_KAMELETS_VERSION=4.0.0 -e YAKS_JBANG_KAMELETS_LOCAL_DIR=../../../kamelets" 
```

Comment:
I have had a look into the failing YAKS tests and I think `-e YAKS_JBANG_KAMELETS_VERSION=4.0.0` is a problem because the v4.0.0 has not been released yet. I can see this in the logs when Camel JBang starts the route:
```
[jbang] Resolving dependencies...
[jbang]    org.apache.camel:camel-bom:4.0.0@pom
[jbang]    org.apache.camel:camel-jbang-core:4.0.0
[jbang]    org.apache.camel.kamelets:camel-kamelets:4.0.0
[jbang] [ERROR] Could not resolve dependencies: Could not find artifact org.apache.camel.kamelets:camel-kamelets:jar:4.0.0 in central (https://repo1.maven.org/maven2)
```

Reply:
Can we keep using `4.0.0-SNAPSHOT` here and bump to 4.0.0 once released?

Reply:
ah understand the issue.
The problem that I see if using 4.0.0-SNAPSHOT is how to be sure that it i suing the local Kamelets so that the tests are launched with the Kamelets of the branch  and not the one from the last published main branch?

Reply:
The env setting `-e YAKS_JBANG_KAMELETS_LOCAL_DIR=../../../kamelets` makes sure to use the local Kamelets from the current checked out branch. Also when the GitHub CI workflow builds the 4.0.0-SNAPSHOT version before running the tests Maven and Camel JBang should also use these fresh built bits form the current branch, too.